specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2280 @@
+        },
+        "startTime": {
+          "description": "The operation start time (ISO 8601 UTC standard).",
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time"
+        },
+        "endTime": {
+          "description": "The operation end time (ISO 8601 UTC standard).",
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time"
+        },
+        "percentComplete": {
+          "type": "number",
+          "format": "double",
+          "minimum": 0,
+          "maximum": 100,
+          "description": "Percentage completed.",
+          "readOnly": true
+        },
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "object",
+            "properties": {}
+          },
+          "x-ms-identifiers": [],
+          "description": "A list of results when the operation returns multiple results."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results.",
+          "readOnly": true
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/OperationResultProperties"
+        },
+        "error": {
+          "$ref": "#/definitions/OperationResultErrorProperties"
+        }
+      }
+    },
+    "Status": {
+      "readOnly": true,
+      "description": "The status of operation.",
+      "type": "string",
+      "enum": [
+        "Succeeded",
+        "Canceled",
+        "Failed",
+        "Running"
+      ],
+      "x-ms-enum": {
+        "name": "Status",
+        "modelAsString": true
+      }
+    },
+    "OperationResultProperties": {
+      "type": "object",
+      "properties": {},
+      "description": "Operation result properties."
+    },
+    "OperationResultErrorProperties": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "The code of the error.",
+          "readOnly": true
+        },
+        "message": {
+          "type": "string",
+          "description": "The error message.",
+          "readOnly": true
+        }
+      },
+      "description": "Operation result error properties."
+    },
+    "ContactProfileThirdPartyConfiguration": {
+      "type": "object",
+      "required": [
+        "providerName",
+        "missionConfiguration"
+      ],
+      "properties": {
+        "providerName": {
+          "type": "string",
+          "description": "Name of the third-party provider."
+        },
+        "missionConfiguration": {
+          "type": "string",
+          "description": "Name of string referencing the configuration describing contact set-up for a particular mission. Expected values are those which have been created in collaboration with the partner network."
+        }
+      }
+    }
+  },
+  "parameters": {
+    "apiVersionParameter": {
```

Comment:
In terms of openAPI authoring best practice, I'd recommend NOT modeling api-version as an enum but instead just referencing ApiVersionParameter:

Common parameters best practice - use common types
There are definitions of common parameters in commontypes types.json that you can use by $ref instead of declaring your own, like SubscriptionIdParameter, OperationIdParameter, and ApiVersionParameter,
See
[https://github.com/Azure/azure-rest-api-specs/blob/be6cd9ccfcb6ba08c1c206627026eabfbff31fc1/specification/common-types/resource-management/v3/types.json#L630](https://github.com/Azure/azure-rest-api-specs/blob/be6cd9ccfcb6ba08c1c206627026eabfbff31fc1/specification/common-types/resource-management/v3/types.json#L630)

Reply:
need a discussion on this change. holding off on this one for now

Reply:
made the changes to use common-types